tasks/install.yml

```diff
@@ -1,4 +1,22 @@
 ---
+- name: Add EPEL repository
+  package:
+    name: epel-release
+    state: latest
+  tags:
+    - skip_ansible_lint
+
+# Around 2017-07-12, installing epel-release stopped being idempotent. The
```

Comment:
Unless someone has a better solution, I'll apply this fix to the other ansible roles that are failing idempotence checks in Jenkins after we merge this PR.